src/pkgcheck/checks/git.py

```diff
@@ -291,10 +305,13 @@ class GitPkgCommitsCheck(GentooRepoCheck, GitCommitsCheck):
             SuspiciousSrcUriChange,
             PythonPEP517WithoutRevbump,
             EAPIChangeWithoutRevbump,
-        ]
+            OldPythonCompat,
+        }
     )
 
     python_pep517_regex = re.compile("^DISTUTILS_USE_PEP517=")
+    python_compat_declare_regex = re.compile(r"^declare -a PYTHON_COMPAT=(?P<value>.+)$")
```

Comment:
Is this going to work where we have PYTHON_TESTED and COMPAT defined in terms of it?

Reply:
Yes, since it uses real bash "run" for the environment computation
I've taken pytest, readded 3.9 to the tested array:
```diff
diff --git a/dev-python/pytest/pytest-7.4.0.ebuild b/dev-python/pytest/pytest-7.4.0.ebuild
index 34cc63f9e93..64772027f76 100644
--- a/dev-python/pytest/pytest-7.4.0.ebuild
+++ b/dev-python/pytest/pytest-7.4.0.ebuild
@@ -4,7 +4,7 @@
 EAPI=8
 
 DISTUTILS_USE_PEP517=setuptools
-PYTHON_TESTED=( python3_{10..12} pypy3 )
+PYTHON_TESTED=( python3_{9..12} pypy3 )
 PYTHON_COMPAT=( "${PYTHON_TESTED[@]}" )
 
 inherit distutils-r1 multiprocessing pypi

```
and got:
```
OldPythonCompat: version 7.4.0: old PYTHON_COMPAT target listed: [ python3_9 ]
```

Reply:
Thank you!